R/addins.R

```diff
@@ -68,7 +68,7 @@ style_active_file <- function() {
   } else if (is_r_file) {
     out <- try_transform_as_r_file(context, transformer)
   } else {
-    abort("Can only style .R, .Rmd and .Rnw files.")
+    abort("Can only style .R, .qmd, .Rmd, and .Rnw files.")
```

Comment:
Can we sort alphabetically? First thought we should also standardise casing, but seems like quarto does not encourage or even work with `.Qmd`.